.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+### config
+conf.js
```

Comment:
useless since a react project doesn't have such a file :)

Reply:
I deleted conf file and deleted conf.js in .gitignore (error of repository :) )

Reply:
they're still in the PR :)